src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -25,10 +25,20 @@
     public static final String CONCUR_CONSUMER_KEY = "concur.consumer.key";
     public static final String CONCUR_SECRET_KEY = "concur.secret.key";
     public static final String CONCUR_ACCESS_TOKEN_EXPIRATION_DATE = "concur.access.token.expiration.date";
-    
+
     public static final String EXPENSE_REPORT_URI_INDICATOR = "/expense/expensereport/";
     public static final String TRAVEL_REQUEST_URI_INDICATOR = "/travelrequest/";
-    
+
     public static final String DEFAULT_TRAVEL_REQUEST_OBJECT_CODE_PARAMETER_NAME = "DEFAULT_TRAVEL_REQUEST_OBJECT_CODE";
 
+    public static final String EXPENSE_AWAIRING_EXTERNAL_VALIDATION_STATUS_CODE = "A_EXTV";
+    public static final String REQUEST_AWAIRING_EXTERNAL_VALIDATION_STATUS_CODE = "Q_EXTV";
+
+    public static final String APPROVE_ACTION = "Approve";
+    public static final String SEND_BACK_TO_EMPLOYEE_ACTION = "Send Back To Employee";
+    public static final String APPROVE_COMMENT = "Approved via Concur Connect";
+    public static final String EXPENSE_WORKFLOW_UPDATE_NAMESPACE = "http://www.concursolutions.com/api/expense/expensereport/2011/03";
```

Comment:
Won't this value have the possibility of changing over time requiring rebuilds and redeploys?  Would this item be better served as a configuration parameter instead of a constant?